napari/_qt/qt_dims.py

```diff
@@ -328,3 +329,128 @@ def focus_down(self):
         else:
             index = (displayed.index(self.last_used) - 1) % len(displayed)
             self.last_used = displayed[index]
+
+    def play_dim(self, axis: int = 0, fps: float = 10, range=None):
+        """
+        Animate (play) axis
+
+        Parameters
+        ----------
+        axis: int
+            Index of axis to play
+        fps: float
+            Frames per second for playback (not guaranteed)
+        range: tuple | list
+            If specified, will constrain animation to [first, last] frames
+
+        Raises
+        ----------
+        IndexError
+            If ``axis`` requested is out of the range of the dims
+        IndexError
+            If ``range`` is provided and out of the range of the dims
+        ValueError
+            If ``range`` is provided and range[0] >= range[1]
+        """
+        # TODO: No access in the GUI yet. Just keybinding.
+        if axis >= len(self.dims.range):
```

Comment:
I think we should also check if `self._displayed_sliders[axis]` is `True`, as these are the axes that actually correspond to sliders, and only proceed if it is `True` and otherwise just stop everything.

Reason is right now I can "play" an axis for a 2D image, nothing bad happens but `is_playing` becomes True and we'll have unnecessary loops in the background

Reply:
fixed